### ISSUE TITLE
logo looks creepy

### ISSUE THREAD

**Issue by jancborchardt** [unhosted/opentabs#5]
The squirrel in the logo creeps me out. Having it German also doesn’t help foreign visitors to http://opentabs.net

The previous version was much more of a punch – like »Terms of Service; Didn’t Read«, with the »banker = wanker« and way better looking. Just take the new text and use the old layout, that’d look a lot better.

cc @michielbdejong

**Comment by jancborchardt** [comment]
Also, for what it’s worth, the squirrel analogy doesn’t really make sense. It seems to be collecting or even amassing coins, which misses the point of Opentabs.

**Comment by melvincarvalho** [comment]
On 15 February 2013 17:58, Jan-Christoph Borchardt <notifications@github.com

> wrote:
> 
> Also, for what it’s worth, the squirrel analogy doesn’t really make sense.
> It seems to be collecting or even amassing coins, which misses the point of
> Opentabs.
> 
> Sort of agree with this.  Need something a bit more punchy.

Just like firefox was 'reclaim the internet' ... we want opentabs to be a
'reclaim your financail power' -- squirrel doesnt do it for me, but hard to
think of what might be more appropriate.

As a side topic, it's interesting to look at money itself.  Bank notes and
coins are incredibly artistic, often borrowing mythological symbolism.  I
wonder if we could take some of that and parody it etc.

**Comment by winterlightning** [comment]
One easy way to create a logo is to just browse the noun project and put a project name under a picture:

http://thenounproject.com/noun/money-transfer/#icon-No3050

I think that conveys the concept in some ways and it's free under CC 3.0.

**Comment by michielbdejong** [comment]
yeah :) i asked the guy to make a squirrel that looks illegal, and he did a wonderful job at that. the German is on purpose, makes it sound more underground.

i'm a bit done with the Opentabs experiments so far, and not going to do much more with this site during probably all of 2013, apart from maybe developing remoteStorage.money a little bit.

if after that we decide to launch a user-facing app, then that can include a website and logo re-design

**Comment by jancborchardt** [comment]
Then so it doesn't look creepy or crappy through all of 2013 when people
visit opentabs.net, can you please rewind it to the previous design (with
the new text though of course.

**Comment by michielbdejong** [comment]
as i said, i like the fact that it's uneasy and disturbing. this is on purpose. so i actually like how it is now, but thanks for the suggestion.

ps are you studying Spanish through your mailclient settings? ;)
